cmd/merge.go

```diff
@@ -7,13 +7,19 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"time"
 
 	"github.com/Clever/microplane/initialize"
 	"github.com/Clever/microplane/merge"
 	"github.com/Clever/microplane/push"
 	"github.com/spf13/cobra"
 )
 
+var mergeFlagThrottle string
+var mergeThrottle string
+
+var mergeLimiter = time.NewTicker(1 * time.Millisecond)
```

Comment:
I tried using the go rate package unsuccessfully. I read the code and it seemed like it should handle multiple goroutines correctly, so I'm not sure :/

This is the same approach as the existing Github rate limiting we're using. I'd be excited about a future PR to switch these over to the rate package, if we can get that working.